### PR TITLE
Ensure approval tests actually run despite using decorators

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

**Review by baierjan — APPROVED** [review]

File: tests/test_approve.py [hunk 1]
```diff
@@ -470,9 +470,11 @@ def test_one_aggr_failed(fake_qem, caplog):
 @responses.activate
 @pytest.mark.parametrize("fake_qem", [("NoResultsError isn't raised")], indirect=True)
 @fake_responses_for_unblocking_incidents_via_openqa_comments(incident=2)
+@fake_openqa_comment_api
```
Comment (mimi1vx) [inline tests/test_approve.py:473]: this should not be decorator but proper pytest fixture ...

File: tests/test_approve.py [hunk 2]
```diff
@@ -481,9 +483,11 @@ def test_approval_unblocked_via_openqa_comment(fake_qem, caplog):
 @responses.activate
 @pytest.mark.parametrize("fake_qem", [("NoResultsError isn't raised")], indirect=True)
 @fake_responses_for_unblocking_incidents_via_openqa_comments(incident=22)
+@fake_openqa_comment_api
```
Comment (mimi1vx) [inline tests/test_approve.py:486]: same as above ....

**Review by mimi1vx — CHANGES_REQUESTED** [review]

**Comment by Martchus** [comment]
Maybe it would still good to merge this for now to get rid of the pipeline failures? Just because I don't know how long it'll take me to refactor this using a pytest fixute. I was already glad to make it work using decorators in general.

**Review by mimi1vx — APPROVED** [review]
for fix of the pipeline ..., but it still needs fix formatting @Martchus

**Pull request merged**